homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -416,13 +429,13 @@ async def async_added_to_hass(self) -> None:
         await super().async_added_to_hass()
         if state := await self.async_get_last_state():
             self._state_ts = state.last_updated
-            if hasattr(state, "attributes") and ATTR_NEXT_STATE in state.attributes:
+            if get_state := state.attributes.get(ATTR_NEXT_STATE):
```

Comment:
This is decently nitpicky, but I feel like `next_state` would be the better name given that you're using `prev_state` down below